.github/workflows/e2e_nightly.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         rm -rf ./* || true
 
     - name: Checkout UR
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      uses: actions/checkout@v4.1.1
```

Comment:
why no hash here...?

Reply:
I missed this after starting without hashes, I'll fix it.

Reply:
perfect, thx!